dbfit-java/core/src/main/java/dbfit/api/DBEnvironment.java

```diff
@@ -189,4 +189,6 @@ void connectUsingFile(String filePath) throws SQLException, IOException,
             FileNotFoundException;
 
     DbStoredProcedureCall newStoredProcedureCall(String name, DbParameterAccessor[] accessors);
+
+    String buildFunctionCall(String procOrFuncName, int numberOfParameters);
```

Comment:
> Let's think about what we're unhappy about it? 

For starters `DbStoredProcedureCall` should be an interface, perhaps, for the vendor API (as well as the other return types in `DBEnvironment`) and the default implmentation could be renamed `DefaultDbStoredProcedureCall` (or similar) but I do want to keep this PR to minimal refactoring of course, and split off anything else to another one.

I'll explore what an approach of changing `DbStoredProcedureCall newStoredProcedureCall(String name, DbParameterAccessor[] accessors)` might look like.